invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/UsersSearch/UsersSearch.js

```diff
@@ -0,0 +1,163 @@
+        currentQueryString={queryString}
+        onInputChange={onInputChange}
+        executeSearch={executeSearch}
+        placeholder={'Search for users'}
+      />
+    );
+  };
+
+  _renderResultsList = results => {
+    return (
+      <div className="results-list">
+        <UsersResultsList
+          results={results}
+          viewDetailsClickHandler={userPid => {
+            const path = generatePath(BackOfficeURLS.userDetails, {
+              userPid: userPid,
+            });
+            this.props.history.push(path);
+          }}
+        />
+      </div>
+    );
+  };
+
+  _renderEmptyResults = (queryString, resetQuery) => {
+    return (
+      <Segment placeholder textAlign="center">
+        <Header icon>
+          <Icon name="search" />
+          No items found!
+        </Header>
+        <div className="empty-results-current">
+          Current search "{queryString}"
+        </div>
+        <Segment.Inline>
+          <ClearButton
+            clickHandler={() => {
+              resetQuery();
+            }}
+          />
+        </Segment.Inline>
+      </Segment>
+    );
+  };
+
+  _renderError = error => {
+    return <IlsError error={error} />;
+  };
+
+  _renderPagination = () => {
+    return <Pagination />;
+  };
+
+  _renderCount = totalResults => {
+    return <div>{totalResults} results</div>;
+  };
+
+  _renderResultsSorting = () => {
+    return config.SORT_BY.length ? (
+      <div className="sorting">
+        <span className="before">Show</span>
+        <ResultsPerPage
+          values={config.RESULTS_PER_PAGE}
+          defaultValue={config.RESULTS_PER_PAGE[0].value}
+        />
+        <span className="middle">results per page sorted by</span>
+        <SortBy
+          values={config.SORT_BY}
+          defaultValue={config.SORT_BY[0].value}
+          defaultValueOnEmptyString={config.SORT_BY_ON_EMPTY_QUERY}
+        />
+        <SortOrder
+          values={config.SORT_ORDER}
+          defaultValue={config.SORT_ORDER[0]['value']}
+        />
+      </div>
+    ) : null;
+  };
+
+  _renderHeader = () => {
+    return (
+      <Grid columns={3} verticalAlign="middle" stackable relaxed>
+        <Grid.Column width={5} textAlign="left">
+          <Count renderElement={this._renderCount} />
+        </Grid.Column>
+        <Grid.Column width={6}>{this._renderPagination()}</Grid.Column>
+        <Grid.Column width={5} textAlign="right">
+          {this._renderResultsSorting()}
+        </Grid.Column>
+      </Grid>
+    );
+  };
+
+  _renderFooter = () => {
+    return (
+      <Grid columns={3} verticalAlign="middle" stackable relaxed>
+        <Grid.Column width={5} />
+        <Grid.Column width={6}>{this._renderPagination()}</Grid.Column>
+        <Grid.Column width={5} />
+      </Grid>
+    );
+  };
+
+  render() {
+    return (
+      <ReactSearchKit
+        searchConfig={{
+          ...apiConfig,
+          url: user.url, // DONT remove the end '/' from user url
```

Comment:
can you comment also where the url is defined because I'm sure someone will remove the slash 😄 and to write the reason in the comment, because we might forget why we should not remove it

Reply:
I am renaming all users to patrons to be consistent so we will have different url and hence we will not collide with invenio_accounts users.